lib/Alchemy/Phrasea/PhraseanetService/Controller/PSExposeController.php

```diff
@@ -189,8 +189,10 @@ public function listPublicationAction(PhraseaApplication $app, Request $request)
         if (!$session->has($passSessionName) && $providerId != null) {
             try {
                 $provider = $this->getAuthenticationProviders()->get($providerId);
-                $session->set($passSessionName, $session->get($provider->getId() . '.provider.access_token'));
-                $session->set($this->getLoginSessionName($request->get('exposeName')), $session->get($provider->getId() . '.provider.username'));
+                if ($provider->getType() == 'PsAuth') {
+                    $session->set($passSessionName, $session->get($provider->getId() . '.provider.access_token'));
+                    $session->set($this->getLoginSessionName($request->get('exposeName')), $session->get($provider->getId() . '.provider.username'));
+                }
```

Comment:
you are not supposed to set/get in session "internal" provider values (prefixed by the provider id), names may change anytime.
you can only get the provider then call getter/setters.
If u need missing infos from the provider we will add it.